astropy_helpers

```diff
@@ -1 +1 @@
-Subproject commit 2f1a7837ff42ce090656f13b9f633f841381eb76
+Subproject commit fd7cca2b3340f8ce968f60da89b8c36b93575918
```

Comment:
well this isn't supposed to be here... did you run a `git commit -a` and get astropy_helpers by mistake!

Can you see if you can rebase this away?